source/features/remove-projects-tab.tsx

```diff
@@ -6,18 +6,21 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '../libs/features';
 
-const addNewProjectLink = onetime(() => {
-	if (pageDetect.isUserProfile()) {
-		// The link already exists on our profile,
-		// and we can't create projects on others' profiles
-		return;
-	}
+export const canUserEditOrganization = (): boolean => pageDetect.isOrganizationProfile() && select.exists('.pagehead-tabs-item[href$="/settings/profile"]');
+
+export const canUserEditRepo = (): boolean => pageDetect.isRepo() && select.exists('.reponav-item[href$="/settings"]');
```

Comment:
```suggestion
```
After https://github.com/fregante/github-url-detection/pull/3

Reply:
Done

Reply:
Released?

Reply:
Uhhh `npm publish` was still waiting for me OTP 😅

Done now